test/units/TEST-64-UDEV-STORAGE.sh

```diff
@@ -227,7 +227,6 @@ testcase_nvme_basic() {
 testcase_nvme_subsystem() {
     local expected_symlinks=(
         # Controller(s)
-        /dev/disk/by-id/nvme-QEMU_NVMe_Ctrl_deadbeef
```

Comment:
can you maybe turn this around, and validate that after the settle this symlink does not exist?

Reply:
Okay; I see the `for i in "${expected_symlinks[@]}"` loop that runs `udevadm wait --settle --timeout=30 "$i"`; perhaps something after that loop that sends some kind of an error if it detects any filenames in `unexpected_symlinks`, which would have `/dev/disk/by-id/nvme-QEMU_NVMe_Ctrl_deadbeef` as the only item?

Reply:
well, don't overcomplicate it. just check after the loop with a simple "test ! -e /dev/disk/by-id/…" that the unnamespaced symlink doesn't exit. i.e. just add one line. And the wait lines on the namespaced symlinks are enough of a wait, simply because devices are generally processed in sorted order.

Reply:
Okay; I was probably trying to think too many steps ahead.  Trying the "test" statement now.